scripts/test-e2e-standalone.sh

```diff
@@ -41,7 +44,16 @@ if [ ! -e "$PROVIDER_KEY_PATH" ]; then
 fi
 
 export MODEL="${MODEL:-gpt-3.5-turbo-1106}"
-envsubst < $(pwd)/tests/config/singleprovider.e2e.template.config.yaml > "$OLS_CONFIG_FILE"
+envsubst < $(pwd)/tests/config/singleprovider.e2e.template.config.yaml > "${OLS_CONFIG_FILE}.tmp"
+
+# If no provider url is being specified, remove the url field from the config yaml
+# so we use the default provider url values.
+if [ -z ${PROVIDER_URL:-} ]; then
+  grep -v url: "${OLS_CONFIG_FILE}.tmp" > "$OLS_CONFIG_FILE"
```

Comment:
@tisnik fyi this is the kind of hack i'm resorting to for now so we can continue to only need a single config template yaml, but the next step is probably going to be to create a config template yaml for each provider.  (I am trying to avoid doing that because it will mean more files to maintain/update as our config format evolves, though i hope it's not going to change too much more in the near future)